packages/server/providers/couchdb.js

```diff
@@ -185,16 +185,16 @@ const couchDb = (app, io, pouchOptions) => {
       res.status(404).send({ msg: 'Wrong Wargame Name', data: null })
     }
 
-    const db = new PouchDB(databaseName, pouchOptions)
+    const db = new CouchDB(couchDbURL(databaseName))
 
     db.find({
       selector: {
         messageType: INFO_MESSAGE,
-        _id: { $ne: wargameSettings }
+        _id: { $ne: wargameSettings, $gte: null }
```

Comment:
But @lilit - at the start of the game, there is just the `initial_wargame` document.  That one _is_ the `latest`.

Reply:
Hmm. The game is saving and updating correctly, but I don't know how it gets the `initial_wargame`.

I ran this query, and nothing was returned.
![image](https://user-images.githubusercontent.com/1108513/181816873-efe2edf3-166e-46cb-b935-9989abdb1460.png)


Aaah, hold on. Maybe we _only_ get the `latest` wargame after it has been initiated - in which case we don't want the initial wargame :-)

Let me try that out on Monday :-)

Reply:
Ok. I've just played.  It doesn't quite work.

In CouchDb we just have `initial_wargame`.  It is at Turn Zero, and `wargame_initiated: false`.

I then enter player view as White - Game Control.

My task is to give the ships their initial positions.  So, I zoom in a bit, then one-by-one select each ship from the "Laydown Pen" and position it.

Once they are all positioned, I click on "Submit Routes", and say "Yes" to the dialog.

Since the wargame is not yet initiated, we should still be overwriting `initial_Wargame`.

The _reducer_ for `force laydown` gets the newest wargame, sets the ship positions, and saves it as a new wargame.

![image](https://user-images.githubusercontent.com/1108513/181818120-adb00551-af83-41cd-b25d-e3ce5f093311.png)

It should have overwritten `initial wargame`.  It's only once `wargame_initiated: true` that we create new version(s).

Reply:
Issue resolved! 

We only get the `latest` if the wargame has been initiated.  It's working fine :-)